public/ticketmanager.php

```diff
@@ -294,102 +325,108 @@
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 7)) . "'>*RA Emulator</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 7)) . "'>RA Emulator</a> | ";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 7)) . "'>RA Emulator</a> | ";
                 }
 
                 if ($rarchKnownTickets) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 8)) . "'>*RetroArch - Defined</a></b> | ";
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 8)) . "'>*RetroArch - Defined</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 8)) . "'>RetroArch - Defined</a> | ";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 8)) . "'>RetroArch - Defined</a> | ";
                 }
 
                 if ($rarchUnknownTickets) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 9)) . "'>*RetroArch - Undefined</a></b> | ";
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 9)) . "'>*RetroArch - Undefined</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 9)) . "'>RetroArch - Undefined</a> | ";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 9)) . "'>RetroArch - Undefined</a> | ";
                 }
 
                 if ($emulatorUnknownTickets) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 10)) . "'>*Emulator Unknown</a></b>";
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 10)) . "'>*Emulator Unknown</a></b>";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 10)) . "'>Emulator Unknown</a>";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 10)) . "'>Emulator Unknown</a>";
                 }
                 echo "</div>";
 
                 // Core/Unofficial Filters - These filters are mutually exclusive
                 echo "<div>";
                 echo "<b>Achievement State:</b> ";
+                $gamesFilterUrl = '/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&p=$reportedByUser&r=$resolvedByUser&f=';
```

Comment:
Still need to convert the single quotes to double quotes so the variables will be expanded.

Reply:
Removed `$gamesFilterUrl` but converted to double quotes in the new Url variables created